larray_editor/editor.py

```diff
@@ -665,6 +674,33 @@ def open_tutorial(self):
     def open_api_documentation(self):
         QDesktopServices.openUrl(QUrl("http://larray.readthedocs.io/en/stable/api.html"))
 
+    def about(self):
+        """About Editor"""
+        versions = get_versions()
+        message = \
+            """<b>LArray Editor {editor}</b>
+            <br>The Graphical User Interface for LArray. This project aims to provide an intuitive tool to 
+            visualize and manipulate labelled N-dimensional arrays. Licensed under the terms of the 
+            GNU GENERAL PUBLIC LICENSE Version 3.
+            <p>Created by Gaëtan de Menten.
+            <br>Developed and maintained by the Federal Planning Bureau (Belgium). 
+            Many thanks to all LArray regular users.
+            <p>For bug reports and feature requests, please go to our <a href="{website}">Github website</a>. 
```

Comment:
I would rather have a direct "Report a bug or suggest a new feature" menu item in the Help menu.